opencxl/cxl/component/virtual_switch/port_binder.py

```diff
@@ -91,30 +94,36 @@ class BIND_STATUS(Enum):
 
 @dataclass
 class BindSlot:
-    vppb_connection: CxlConnection
+    vppb: Vppb
     status: BIND_STATUS = BIND_STATUS.INIT
     processor: Optional[BindProcessor] = None
     dsp: Optional[DownstreamPortDevice] = None
 
 
 class PortBinder(RunnableComponent):
-    def __init__(self, vcs_id: int, vppb_connections: List[CxlConnection]):
+    def __init__(self, vcs_id: int, vppbs: List[Vppb]):
         super().__init__()
         self._vcs_id = vcs_id
-        self._vppb_connections = vppb_connections
+        self._vppbs = vppbs
         self._bind_slots: List[BindSlot] = []
         self._async_gatherer = AsyncGatherer()
-        for vppb_connection in self._vppb_connections:
+        for vppb in self._vppbs:
             bind_slot = BindSlot(
-                vppb_connection=vppb_connection,
+                vppb=vppb,
             )
             self._bind_slots.append(bind_slot)
 
+        self._dummy = BindProcessor(self._vcs_id, 0, CxlConnection(), CxlConnection())
```

Comment:
This is actually indeed a dummy instance. This is because we need at least one BindProcessor running so that PortBinder's AsyncGatherer doesn't end prematurely.

When we implement dynamic binding, this will be removed and BindProcessor(s) will be created/removed dynamically with bind_vppb()/unbind_vppb().

We'll keep this as-is with a comment clearly indicating why this is needed and what'll happen next.